velox/common/memory/MappedMemory.h

```diff
@@ -205,6 +195,7 @@ class MappedMemory : public std::enable_shared_from_this<MappedMemory> {
 
    private:
     uint64_t data_;
+    MachinePageCount numPages_;
```

Comment:
Any particular reason we must increase size of this struct?  Could cause performance degradation.
cc: @oerling